opendaq/Opc.Ua.Daq.Bsp.NodeSet2.xml

```diff
@@ -181,8 +181,8 @@
         <DisplayName>DaqAxisInformation</DisplayName>
         <References>
             <Reference ReferenceType="HasTypeDefinition">i=69</Reference>
-            <Reference ReferenceType="HasDescription" IsForward="false">ns=1;i=5024</Reference>
```

Comment:
One question: Could you modify the file in that way, that these things are not shown as changes.